app/components/deadline_banner_component.html.erb

```diff
@@ -1,9 +1,14 @@
 <% if CycleTimetable.show_apply_1_deadline_banner? %>
   <%= govuk_notification_banner(title: "Important") do |notification_banner| %>
-    <%= notification_banner.slot(:heading, text: "Apply now to get on a course starting in the #{CycleTimetable.cycle_year_range} academic year") %>
-    <p class="govuk-body">Courses can fill up at any time, so you should apply as soon as you can.</p>
-    <p class="govuk-body">If you’re applying for the first time since applications opened in <%= CycleTimetable.find_opens.to_s(:month_and_year) %>, apply no later than <%= apply_1_deadline %>.</p>
-    <p class="govuk-body">If your application did not lead to a place and you’re applying again, apply no later than <%= apply_2_deadline %>.</p>
+    <%= notification_banner.slot(:heading, text: "If you’re applying for the first time since applications opened in October 2020") %>
+    <p class="govuk-body">It’s no longer possible to apply for teacher training starting in the <%= CycleTimetable.cycle_year_range %> academic year.</p>
+    <p class="govuk-body">You can <%= govuk_link_to('start or continue your application', Settings.apply_base_url) %> to get it ready for courses starting in the <%= CycleTimetable.cycle_year_range(CycleTimetable.next_year) %> academic year.</p>
+    <p class="govuk-body">
+      You’ll be able to find courses from <%= CycleTimetable.find_reopens.to_s(:govuk_time) %> on <%= CycleTimetable.find_reopens.to_s(:govuk_date) %> and submit your application from <%= CycleTimetable.apply_reopens.to_s(:govuk_time) %> on <%= CycleTimetable.apply_reopens.to_s(:govuk_date) %>.
```

Comment:
so i tweaked the date_and_time method on the Find codebase so it fits with this style 

![image](https://user-images.githubusercontent.com/42515961/129929607-ecfdce94-5ddc-4444-9e9c-e187f1d9654f.png)

I think we can just use that